src/libsyntax/ext/expand.rs

```diff
@@ -440,8 +440,7 @@ fn expand_annotatable(mut item: Annotatable, fld: &mut MacroExpander) -> SmallVe
                 callee: NameAndSpan {
                     format: MacroAttribute(intern(&attr.name())),
                     span: Some(attr.span),
-                    // attributes can do whatever they like, for now
-                    allow_internal_unstable: true,
+                    allow_internal_unstable: false,
```

Comment:
I believe so, yes, but only a plugin-breaking-change. That is, all existing plugins today could silently rely on unstable features, but now they won't be able to unless they manufacture one of these spans.

Note, though, that if we *don't* change this then all `rustc-macro` derive modes would also be able to rely on unstable features :(